RDD with Discrete Running Variables.md

```diff
@@ -0,0 +1,134 @@
+
+## Using local randomisation with few distinct mass points
+
+As previously mentioned, identification in the local randomisation approach rests on the formal assumption that, within a certain sufficiently narrow bandwidth bounded by $c-h$ and $c+h$, where $c$ is the cutoff value, units just above and just below the cutoff are fundamentally similar and comparable to each other, and that therefore treatment assignment based on the cutoff value is as if random. This implies that the running variable is random and orthogonal to the values of the outcome variable.  
+
+When applying local randomisation, we estimate the difference in means between the treatment and control groups. That is, if treatment is assigned above the cutoff, the treatment effect theta is the mean outcome of all observations above the cutoff minus the mean outcome of all observations below the cutoff. Thus, we are not estimating the treatment effect at the cutoff, but rather within the entire bandwidth or window we are using.  
+
+{{% tip %}}
+Read more about the local randomisation approach [here](https://tilburgsciencehub.com/topics/analyze/causal-inference/rdd/local-randomization/).
+{{% /tip %}}
+
+The advantage of the local randomisation design when the running variable is discrete is that it does not rely on assumptions about the continuity of the regression functions on either side of the cutoff. This is relevant because when the running variable is discrete and there are large gaps between the mass points, it is often implausible to assume that the conditional expectation function (CEF) relating the running variable to the outcome is continuous. We are also unable to observe the nature of the CEF within each mass point. For example, if our running variable is income grouped by ranges (e.g., $24,000-$24,999.99, $25,000-$25,999.99 etc.), we can observe values of the outcome for each range, but not for each individual subject within that range. That is, we cannot discern the relationship between income and the outcome for individuals earning between $24,000 and $24.999.99. By lifting the continuity assumptions, the local randomisation approach allows us to study treatment effects in RD designs where the assignment variable is discrete, low-frequency, and sparse.  

```

Comment:
Also add a subtitle Advantages of this approach or something alike for better structuring